content/messages/messages-en.yaml

```diff
@@ -0,0 +1,116 @@
+more_curious: "A bit more curious?"
+changelog_of_v2: "Changelog of the v2.0"
+webinars_of_v2: "Webinars of the v2.0"
+userguides_for_v16_17: "Userguides for v1.6 & v1.7"
+marketplace: "Marketplace"
+customer_stories: "Customer stories"
+knowledge_base: "Knowledge base"
+coding: "Coding right! But what exactly?"
+build_connectors: "Build connectors,<br>buddy!"
+api_friend: "The API is your friend"
+api_friend_description: "Connect your PIM with ERP, DAM, print solutions, ecommerce platforms, the world... Everything is possible!"
+go_API: "Browse the API website"
+pimp_my_pim: "Pimp my PIM,<br>baby!"
+follow_cookbooks: "Follow your cookbooks"
+follow_cookbooks_description: "Install a brand new PIM, migrate to a new version, customize, dig deeper in our technical stack... Create awesome things!"
+technical_docs: "Browse the technical docs"
+search_for_anything: "Search for anything..."
+
+complete_guides: "The complete guides to master previous PIM versions"
+end_user: "End-user"
+download_pdf: "Download the PDF"
+catalog_settings: "Catalog settings"
+administrator: "Administrator"
+end_of_user_guides: "I guess you reached the end!<br>Still looking for help? Join the community on"
+versions_below: "The versions below are not supported by our team anymore."
+run_their_course: "It's not that we don't like them, it's just that they have run their course and now, they need to rest a little."
+
+found_a_typo: "Found a typo or a hole in the documentation and feel like contributing?"
+join_us_on_github: "Join us on Github!"
+
+browse_all_our_themes: "Browse topics"
+
+using_akeneo_theme: "Using Akeneo"
+using_akeneo_description_theme: "You are our dear Julia: the PIM is your daily tool. You want to be productive and have the best products catalog ever. We're here to help you."
+setting_up_the_pim_theme: "Setting up the PIM"
+setting_up_the_pim_description_theme : "Having trouble with your PIM catalog structure? Don't know how to set up permissions? Not to worry, you have come to the right place!"
+discover_akeneo_concepts: "Discover Akeneo concepts"
+discover_akeneo_concepts_description: "New to Akeneo? Start here with the basics. Remember every Jedi Knight began as a young Padawan!"
+build_up_your_catalog: "Building your catalog"
+build_up_your_catalog_description: "Tailoring the catalog model to the needs of your users is absolutely crucial. Before beginning to play with Akeneo, be sure to check out these few key settings."
+users_and_permissions: "Users and permissions"
+users_and_permissions_description: "As an administrator, you will need to manage all the users that will access the PIM. Here are some articles to help you understand how to handle the users and their permissions."
+languages_and_translations: "Languages and translations"
+languages_and_translations_description: "We love being international here. The more languages the PIM will speak, the happier we will be. There is already a lot of happiness at Akeneo, but let's bring some more!"
```

Comment:
the PIM will offer (instead of speak, because a PIM does not really speak right :p!)